packages/components/ContextualMenu/src/SubmenuItem.tsx

```diff
@@ -119,6 +119,19 @@ export const SubmenuItem = compose<SubmenuItemType>({
       [onItemHoverIn],
     );
 
+    const chevronSvg: React.FunctionComponent<SvgProps> = (props: SvgProps) => {
+      return (
+        <Svg width={props.width} height={props.height} viewBox="0 0 2048 2048" color={props.color}>
+          <G transform={I18nManager.isRTL ? 'translate(2048, 0) scale(-1, 1)' : ''}>
+            <Path fill="currentColor" d="M 743 1767 l -121 -121 l 708 -707 l -708 -708 l 121 -121 l 828 829 z"></Path>
+          </G>
+        </Svg>
+      );
+    };
+    const svgProps: SvgIconProps = {
+      src: chevronSvg,
+    };
```

Comment:
Thanks for this fix! I really like the encapsulation into an IconProps to be really nice :) 
Out of scope of this PR, I wonder: How would you feel if we introduced something like a "Flips in RTL" prop to Icon in the future, so that Icon would then be in charge of checking `I18nManager` and adding a transform? I'm not sure it's feasible with SVGs but it's been on my mind a bit as a generic solution.